public/client.js

```diff
@@ -0,0 +1,133 @@
+var ViewControlsDisplay =  Backbone.View.extend({/*properties*/});
+var viewControlsDisplay = new ViewControlsDisplay({el: '#controlsdisplay'});
+
+var ViewListDisplay =  Backbone.View.extend({/*properties*/});
+var viewListDisplay = new ViewListDisplay({el: '#catalogue'});
+
+var ViewAddDisplay = Backbone.View.extend({/*properties*/});
+var viewAddDisplay = new ViewAddDisplay({el: '#adddisplay'});
+
+var rcRecord = new Backbone.Model.extend({
+  initialize: function(){console.log("rcRecord initialized")}
+              });
+var newRcRecord = new RcRecord();
```

Comment:
You defined your model constructor as rcRecord (line 10) but you're calling it RcRecord here.